docs/excel/custom-functions-overview.md

```diff
@@ -353,10 +353,10 @@ function refreshTemperature(thermometerID){
 }
 ```
 
-## Co-Authoring
+## Coauthoring
 Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature works with custom functions. If your workbook uses a custom function, your colleague will be prompted to load the custom function's add-in. Once you both have loaded the add-in, the custom function will share results through co-authoring.
 
-For more information on co-authoring, see [About Co-Authoring in Excel](https://docs.microsoft.com/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).
+For more information on co-authoring, see [About Co-Authoring in Excel](/en-us/office/vba/excel/concepts/about-coauthoring-in-excel).
```

Comment:
- remove `/en-us` from beginning of URL
- the link text should match the title of the article that it links to -> "Coauthoring in Excel add-ins"
- since the VBA article about coauthoring spells the term without a hyphen, recommend that we do the same throughout our doc set -- i.e., use "coauthoring" (without the hyphen)